### PR TITLE
Fix build issues

### PR DESCRIPTION
**Pull request by ahmetoz**
## Hints for the review
- The issue below solved with which was causing the main issues https://github.com/commercetools/commercetools-payone-integration/pull/294/commits/056636201fca3521d3cfc555fb2e716244659a2a
````
Installing Google Chrome stable
0.30s$ wget --no-verbose -O /tmp/$(basename $CHROME_SOURCE_URL) $CHROME_SOURCE_URL
2020-08-10 10:36:46 URL:https://dl.google.com/dl/linux/direct/google-chrome-stable_current_amd64.deb [69636412/69636412] -> "/tmp/google-chrome-stable_current_amd64.deb" [1]
dpkg-deb: error: archive '/tmp/google-chrome-stable_current_amd64.deb' has premature member 'control.tar.xz' before 'control.tar.gz', giving up
dpkg: error processing archive /tmp/google-chrome-stable_current_amd64.deb (--install):
 subprocess dpkg-deb --control returned error exit status 2
Errors were encountered while processing:
 /tmp/google-chrome-stable_current_amd64.deb
````
- chromedriver was in 79 and the stable version is 84, so it's updated.

- guava library causing the issues on the build are reverted, I am not %100 sure why it causes timeouts (10 minute which causes travis build fails) 

- Redirect URL from 3ds verification is different than in tests, so it's updated

## Todos
- For the issues related to payment without 3ds are asked to pay one support, so it's not supported we will deactivate those tests (remove it's possible)

- Revert concordion library to v2 again

- Add exclude for guava library to avoid depandabot commits

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -36,10 +36,10 @@ def depVersions = [
         mockito: '3.3.3',
 
         // : functionaltests
-        concordion: '2.2.0',
+        concordion: '1.5.1',
```
Comment (butenkor) [inline build.gradle:39]: Why do we downgrade here? Did not we say that issue is not related to concordion version?

Comment (ahmetoz) [reply build.gradle:39]: @JudeNiroshan could you check the todo section of the PR ?

Comment (JudeNiroshan) [reply build.gradle]: 7202712

Comment (ahmetoz) [reply build.gradle]: what about concordionParallelRunExtension ?

File: build.gradle [hunk 2]
```diff
@@ -39,7 +39,7 @@ def depVersions = [
         concordion: '2.2.0',
         concordionInputStyleExtension: '0.1',
         concordionConcordionRunTotalsExtension: '1.2.0',
-        concordionParallelRunExtension: '1.1.0',
+        concordionParallelRunExtension: '1.0.1',
```
Comment (ahmetoz) [inline build.gradle:42]: This could be reverted to 1.1.0 too.

File: .travis.yml [hunk 3]
```diff
@@ -6,6 +6,9 @@ services:
   - docker
 addons:
   chrome: stable
+  apt:
+    packages:
+      - dpkg
```
Comment (butenkor) [inline .travis.yml:11]: Why do we need it suddenly?

File: functionaltests/src/test/java/util/CustomWebDriver.java [hunk 4]
```diff
@@ -14,7 +14,7 @@
     private ChromeDriver driver;
 
     CustomWebDriver() {
-        String currentDriver = SystemUtils.IS_OS_MAC ? "chromedriver_mac_85" : "chromedriver_linux_85";
+        String currentDriver = SystemUtils.IS_OS_MAC ? "chromedriver_mac_84" : "chromedriver_linux_84";
```
Comment (butenkor) [inline functionaltests/src/test/java/util/CustomWebDriver.java:17]: Is it bacause Travis is using older version?

File: build.gradle (hunk 2 above)
Comment (JudeNiroshan) [reply build.gradle]: 1e58a47

File: build.gradle (hunk 1 above)
Comment (JudeNiroshan) [reply build.gradle]: 1e58a47

File: .travis.yml (hunk 3 above)
Comment (ahmetoz) [reply .travis.yml:11]: check the hints for the review section, the old ubuntu having issues with dpkg so I added it with apt addon, 
also another option could be using the default ubuntu setup:

https://docs.travis-ci.com/user/trusty-to-xenial-migration-guide
but could not have time to migrate it correctly, it failed, maybe @JudeNiroshan could handle if he has time


```
Installing Google Chrome stable
0.30s$ wget --no-verbose -O /tmp/$(basename $CHROME_SOURCE_URL) $CHROME_SOURCE_URL
2020-08-10 10:36:46 URL:https://dl.google.com/dl/linux/direct/google-chrome-stable_current_amd64.deb [69636412/69636412] -> "/tmp/google-chrome-stable_current_amd64.deb" [1]
dpkg-deb: error: archive '/tmp/google-chrome-stable_current_amd64.deb' has premature member 'control.tar.xz' before 'control.tar.gz', giving up
dpkg: error processing archive /tmp/google-chrome-stable_current_amd64.deb (--install):
 subprocess dpkg-deb --control returned error exit status 2
Errors were encountered while processing:
 /tmp/google-chrome-stable_current_amd64.deb
```

File: functionaltests/src/test/java/util/CustomWebDriver.java (hunk 4 above)
Comment (ahmetoz) [reply functionaltests/src/test/java/util/CustomWebDriver.java:17]: I reverted the broken change that is added by Andi he added v85 which is beta, travis chrome addon has 2 options beta and stable, stable is 84.

in desc I said: `chromedriver was in 79 and the stable version is 84, so it's updated.`

Comment (JudeNiroshan) [reply functionaltests/src/test/java/util/CustomWebDriver.java:17]: chromedriver 84 is the latest stable version that chrome is using. 85 is beta release

Comment (ahmetoz) [reply functionaltests/src/test/java/util/CustomWebDriver.java:17]: for more details: https://docs.travis-ci.com/user/chrome#selecting-a-chrome-version

Comment (butenkor) [reply functionaltests/src/test/java/util/CustomWebDriver.java:17]: clear. Thanks

**Comment by JudeNiroshan** [comment]
@butenkor I have added an external library to maintain the chromedriver. Therefore we do not have to update it time to time. https://github.com/bonigarcia/webdrivermanager
